TigranKirakosov/hw4.js

```diff
@@ -16,7 +16,6 @@ Order.prototype.deleteFromOrder = function(position) {
     return posToKeep !== position;
   }));
 };
-// item should be a string: ex. 'size', 'stuffing', 'type'

```

Comment:
Actually, there were supposed to be more fixes, but Prettier plugin made the work and now I see that these fixes were unnecessary